client/packages.go

```diff
@@ -112,6 +114,42 @@ func (client *Client) List(names []string) ([]*Snap, error) {
 	return result, nil
 }
 
+func setPrice(snap *Snap, currency string) {
```

Comment:
Yeah, it may have been neater to have it as a method. If any one else in @snapcore/snapd-committers has an option I'm all ears :)